invenio_preservation_sync/services/schemas.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-Preservation-Sync is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Schema class for the Preservation Info object."""
+
+from datetime import timezone
+
+from marshmallow import Schema, fields
+from marshmallow_utils.fields import TZDateTime
+
+from ..models import PreservationStatus
+
+
+class PreservationInfoSchema(Schema):
+    """Schema for a Preservation Info object."""
+
+    pid = fields.String(required=True)
+    revision_id = fields.Integer(required=True)
```

Comment:
align with the data model

Reply:
In the RFC we defined this as a mandatory field.